### ISSUE TITLE
Mouse clicking a Y/N prompt on local tiles doesn't work

### ISSUE THREAD

**Issue by Goratrix** [crawl/crawl#3074]
In current trunk crawl_tiles-0.30-a0-989-g0d87f670db version on Windows 10, left-clicking this prompt doesn't work, it seems like some other character is being sent by the click:

![image](https://user-images.githubusercontent.com/54112839/229584979-1507d044-2631-4539-9221-22f1f72896c1.png)

**Comment by rawlins** [comment]
In current trunk, this works on mac tiles, can you update if this still replicates? (Also, I was getting to this popup by using `w` from the item description, it's possible there are other ways that could behave differently.)

**Comment by Goratrix** [comment]
Thanks! Yeah I tested it now in wizmode with distortion weapon and also Drain weapons and armor and it works fine. Weird thing is, I could swear I saw the bug again about a week ago. If I see it again I'll ping you and will try to notice any specific circumstances.

**Comment by Goratrix** [comment]
Ok, I just had it happen again, and I can replicate it from the save file (attached) but not always??? With the same sequence of keys it sometimes works and sometimes not. Open the save file, press 'i' for inventory, 'z' to select faith amulet (or select amulet with mouse, doesn't seem to matter), prompt comes up, clicking 'Y' with mouse sometimes works and sometimes not. Tried it (reloading the same save) about 20 times, half of time it worked fine.

[Goratrix.zip](https://github.com/crawl/crawl/files/12386079/Goratrix.zip)

![image](https://github.com/crawl/crawl/assets/54112839/03dba105-776e-40ec-9a0a-361f825c1ff6)

**Comment by rawlins** [comment]
Thanks! I still can't replicate this on mac, even with these instructions / save, so it's either windows-specific or there's another missing factor. (I do have access to a windows device now that I can check this on, sooner or later.)

Also, just to double check, are you doing this on current trunk? b56e7cad850ba7 fixed some issues in how menus that required capital letters worked.

**Comment by Goratrix** [comment]
Yes, it's happening in current trunk, right now I got it in 0.31-a0-604-g8e0283293c. Here it is again, it's happening but without any discernible pattern:

![image](https://github.com/crawl/crawl/assets/54112839/b7e8ce88-4a29-4950-a81a-f5ecd7138365)
